### PR TITLE
Review README

### PR DESCRIPTION
**Pull request by pmatulis**
There is no bug tracker set up in Launchpad so I have included the standard Bugs section but left it commented out.

### CODE REVIEW

**Comment by pmatulis** [comment]
The VMWare integration section will likely be moved to the dedicated Ceph docs once they exist.

File: README.md [hunk 1]
```diff
@@ -1,233 +1,160 @@
...
-```bash
-   $ juju ssh ceph-iscsi/1
-   $ sudo gwcli
-   /> ls
-```
+We are assuming a pre-existing Ceph cluster.
+
+To provide multiple data paths to clients deploy exactly two ceph-iscsi units:
+
+    juju deploy -n 2 cs:~gnuoy/ceph-iscsi-08
```
Comment (gnuoy) [inline README.md]: Can we change this to cs:~openstack-charmers-next/ceph-iscsi ?

**Review by gnuoy — CHANGES_REQUESTED** [review]
One small nit

File: README.md (hunk 1 above)
Comment (pmatulis) [reply README.md]: Yep,

**Review by gnuoy — APPROVED** [review]
Thanks for the updates

**Pull request merged**